mne/viz/_browser.py

```diff
@@ -0,0 +1,516 @@
+                _picks = np.where(np.in1d(picks, self.mne.picks_data))[0]
+                if len(_picks) == 0:
+                    break
+                this_data = data[_picks, _start:_stop]
+                if isinstance(self.mne.filter_coefs, np.ndarray):  # FIR
+                    this_data = _overlap_add_filter(
+                        this_data, self.mne.filter_coefs, copy=False)
+                else:  # IIR
+                    this_data = _filtfilt(
+                        this_data, self.mne.filter_coefs, None, 1, False)
+                data[_picks, _start:_stop] = this_data
+        # scale the data for display in a 1-vertical-axis-unit slot
+        this_names = self.mne.ch_names[picks]
+        this_types = self.mne.ch_types[picks]
+        stims = this_types == 'stim'
+        white = np.logical_and(np.in1d(this_names, self.mne.whitened_ch_names),
+                               np.in1d(this_names, self.mne.info['bads'],
+                                       invert=True))
+        norms = np.vectorize(self.mne.scalings.__getitem__)(this_types)
+        norms[stims] = data[stims].max(axis=-1)
+        norms[white] = self.mne.scalings['whitened']
+        norms[norms == 0] = 1
+        data /= 2 * norms[:, np.newaxis]
+        self.mne.data = data
+        self.mne.times = times
+
+    def _get_epoch_num_from_time(self, time):
+        epoch_nums = self.mne.inst.selection
+        return epoch_nums[np.searchsorted(self.mne.boundary_times[1:], time)]
+
+    def _redraw(self, **kwargs):
+        """This is usually not necessary for the pyqtgraph-backend as
+        the redraw of objects is often handled by pyqtgraph internally."""
+        pass
+
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+    # INTERACTION
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    @abstractmethod
+    def close_all(self):
+        pass
+
+    @abstractmethod
+    def close_event(self, target):
+        pass
+
+    @abstractmethod
+    def get_n_windows(self):
+        pass
+
+    @abstractmethod
+    def press_key(self, key, target):
+        pass
+
+    @abstractmethod
+    def fake_click(self, point, target, axis, xform, button, kind):
+        pass
+
+    @abstractmethod
+    def click_ch_name(self, ch_index, button):
+        pass
+
+    @abstractmethod
+    def test_childs(self, key, attr):
+        pass
+
+
+def _reload_backend(backend_name):
+    global backend
+    backend = importlib.import_module(name=_backend_name_map[backend_name],
+                                      package='mne.viz')
+    logger.info(f'Using {backend_name} as 2d backend.\n')
+
+
+def _get_browser(inst, **kwargs):
+    """Instantiate a new MNE browse-style figure."""
+    from .utils import _get_figsize_from_config
+    figsize = kwargs.pop('figsize', _get_figsize_from_config())
+
+    _get_browser_backend()
+    browser = backend._init_browser(inst, figsize, **kwargs)
```

Comment:
the way this is done now (passed to the same function as the rest of `kwargs`), it's no longer clear why figsize gets `pop`ped.  Perhaps 

```
default_kwargs = dict(figsize=_get_figsize_from_config())
default_kwargs.update(kwargs)
...
_init_browser(inst, **default_kwargs)
```
WDYT?

Reply:
Yes, you are right, I also would like more if just the kwargs would be passed on to _init_browser